Testscripts/Linux/dpdkTestPmd.sh

```diff
@@ -72,8 +72,25 @@ runTestPmd()
 	cores=1
 	ssh "${server}" "mkdir -p $LOGDIR"
 	ssh "${server}" "mkdir -p  /mnt/huge; mkdir -p  /mnt/huge-1G; mount -t hugetlbfs nodev /mnt/huge && mount -t hugetlbfs nodev /mnt/huge-1G -o 'pagesize=1G' && echo 4096 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages && echo 1 > /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages && grep -i hug /proc/meminfo"
-	mkdir -p  /mnt/huge; mkdir -p  /mnt/huge-1G; mount -t hugetlbfs nodev /mnt/huge && mount -t hugetlbfs nodev /mnt/huge-1G -o 'pagesize=1G' && echo 4096 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages && echo 1 > /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages && grep -i hug /proc/meminfo 
+	mkdir -p  /mnt/huge; mkdir -p  /mnt/huge-1G; mount -t hugetlbfs nodev /mnt/huge && mount -t hugetlbfs nodev /mnt/huge-1G -o 'pagesize=1G' && echo 4096 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages && echo 1 > /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages && grep -i hug /proc/meminfo
 
+	vf_name_client=$(ssh "${client}" ". utils.sh && get_vf_name 'eth1'")
+	pci_info_client=$(ssh "${client}" "ethtool -i $vf_name_client | grep bus-info |  cut -d' ' -f2-")
+	LogMsg "pci_info_client $pci_info_client"
+	vf_name_server=$(ssh "${server}" ". utils.sh && get_vf_name 'eth1'")
+	pci_info_server=$(ssh "${server}" "ethtool -i $vf_name_server | grep bus-info |  cut -d' ' -f2-")
+	LogMsg "pci_info_server $pci_info_server"
+	trx_rx_ips=$(Get_Trx_Rx_Ip_Flags "${server}")
+	if [ ${pmd} = "netvsc" ]; then
+		. ${DPDK_UTIL_FILE} && NetvscDevice_Setup "${server}"
+		. ${DPDK_UTIL_FILE} && NetvscDevice_Setup "${client}"
+		vdev=''
+	elif [ ${pmd} = "failsafe" ];then
+		vdev="--vdev=net_vdev_netvsc0,iface=eth1,force=1"
+	else
```

Comment:
I did put there a check for failsafe as we can not assume that. It helps in code reading/maintaining.
I don't have any plans to add another driver for now, but in the future it could be easily spotted.

Reply:
acked